src/default-xpring-client.ts

```diff
@@ -105,10 +113,10 @@ class DefaultXpringClient implements XpringClientDecorator {
       )
     }
 
-    const account = new AccountAddress()
+    const account = this.networkClient.AccountAddress()
```

Comment:
Spoke offline, we'll leave this a WIP.

Reply:
Brief Explanation of this for posterity:

When calling the network client calls we must send the request in the type format of the client. (Node or Web)

Instead of having these as `new AccountAddress()` where AccountAddress is a static type we call a method on the networkClient to return a new `AccountAddress` of the type of the network client.